### ISSUE TITLE
clarification needed on whether rendition:* are part of the default vocab

### ISSUE THREAD

**Issue by GoogleCodeExporter** [w3c/epub-specs#465]
```
4.2.2 Default Vocabulary says

 * The default vocabulary is a vocabulary (…) whose terms MUST always be unprefixed
 * The Package Metadata Vocabulary is defined to be the default vocabulary for (…)

4.3 Package Metadata Vocabulary says

 * The properties defined in this vocabulary are referenceable using the base IRI http://idpf.org/epub/vocab/package/#. (informative, 4.3.1)
 * then includes all the “rendition:*” properties


Matt: "my understanding while merging FXL is that we were integrating the 
properties into Publications but not specifically making them part of the 
default vocabulary."
```

Original issue reported on code.google.com by `rdeltour@gmail.com` on 1 Sep 2014 at 4:08

**Comment by GarthConboy** [comment]
Spent a little time looking at this issue -- it has to do with the declaration of the "rendition:" prefix.   This was required to be included in EPUB 3.0 and is implied to be present in EPUB 3.0.1.  Epubcheck version 3 validates against EPUB 3.0 and issues errors if the prefix declaration is NOT present.  Epubcheck version 4 validates against EPUB 3.0.1 and issues a warning if the prefix IS present.

This implies to me that the "rendition:" prefixed properties are in the default vocabulary and text for the definition of this vocabulary (unprefixed) should be cleaned up when resolving "https://github.com/IDPF/epub-revision/issues/720".

**Comment by GarthConboy** [comment]
Chatted with Matt -- going to close this issue as duplicate of "https://github.com/IDPF/epub-revision/issues/720" -- it will be resolved with those changes.
